elastic/kibana
# [ResponseOps][Rules] Hide the "Role visibility" dropdown in the new rule form in serverless

Fixes #199642

## Summary

~~This PR hides the role visibility dropdown in the new rule form when in serverless.~~

This PR hides the role visibility dropdown in the new rule form **when only one consumer is available**.

## How to test

1. Run Kibana security serverless and confirm the rules in stack management do not have the role visibility dropdown.
2. Please also make sure that the drop-down still shows when needed(outside of serverless).
